indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/SimpleWorkerProvisioningStrategy.java

```diff
@@ -103,8 +104,8 @@ public Provisioner makeProvisioner(WorkerTaskRunner runner)
     private final Set<String> currentlyTerminating = Sets.newHashSet();
 
     private int targetWorkerCount = -1;
-    private DateTime lastProvisionTime = new DateTime();
-    private DateTime lastTerminateTime = new DateTime();
+    private DateTime lastProvisionTime = DateTimes.nowUtc();
+    private DateTime lastTerminateTime = lastProvisionTime;
```

Comment:
When "now" DateTime objects are created one after another, they almost always will be identical, but sometimes, very rarely, they could differ. It creates a perfect ground for bugs and transient failures, e. g. if those date times are compared for equality, it may always pass tests, but rarely fail in production. In this class it is not yet important, because `lastProvisionTime` and `lastTerminateTime` are not compared in any way.